packages/block-editor/src/components/block-list-appender/index.js

```diff
@@ -22,6 +22,7 @@ function BlockListAppender( {
 	canInsertDefaultBlock,
 	isLocked,
 	renderAppender: CustomAppender,
+	__experimentalPreventAppenderFocus,
```

Comment:
It's not specific to this PR but I'm seeing the trend of adding experimental props/APIs a lot as a way to escape backward compatibility concerns. And sometimes we do this to avoid thinking a lot about the implications and what would it take to do the stable API right (documentation, dev notes, deprecations...) (I also did that some times so I include myself in this trend)

This is concerning a little bit as for most of these, there doesn't seem to be a plan to how/when we make these stable? Can we stop doing this unless we really think it's something subject to change? We should force ourselves to think about the APIs more.

Maybe we could solve this by making a comment in the code to explain why this has been made experimental and how to we plan to make it stable?

Reply:
> there doesn't seem to be a plan to how/when we make these stable?

I assumed there _was_ a plan!

I feel this needs to be discussed in next Core Editor meeting. Do you agree?

Perhaps we could have some automation which can tell us how long a prop has been `__experimental` and highlight those which should be considered for prime time.

> Maybe we could solve this by making a comment in the code to explain why this has been made experimental and how to we plan to make it stable?

Once the comment has is this a blocker for this PR?

Reply:
No, not a blocker when documented.